mmdet/datasets/utils.py

```diff
@@ -133,7 +133,11 @@ def _check_head(self, runner):
             for name, module in model.named_modules():
                 if hasattr(module, 'num_classes') and not isinstance(
                         module, (RPNHead, VGG, FusedSemanticHead, GARPNHead)):
-                    assert module.num_classes == len(dataset.CLASSES), \
+                    if type(dataset.CLASSES) is str:
```

Comment:
dataset.CLASSES must be a tuple of str and should not be str. Otherwise, it will cause errors in other places.

Reply:
@ZwwWayne i updated the pr with a warning when dataset.CLASSES is a str instead of tuple of str